kubernetes/namespaces/monitoring/alerts/alerts.d/nginx.yaml

```diff
@@ -21,7 +21,7 @@ groups:
       description: "Rate of {{ $labels.status }} errors is {{ $value | humanizePercentage }} on service `{{ $labels.service }}`"
 
   - alert: nginx/p99-timing
-    expr: histogram_quantile(0.99, sum by(host, service, le) (rate(nginx_ingress_controller_request_duration_seconds_bucket{service!~"(grafana|metabase|prestashop-svc)"}[5m]))) > 3 and on(service) increase(nginx_ingress_controller_requests[5m]) > 10
+    expr: histogram_quantile(0.99, sum by(host, service, le) (rate(nginx_ingress_controller_request_duration_seconds_bucket{service!~"(grafana|metabase)"}[5m]))) > 3 and on(service) increase(nginx_ingress_controller_requests[5m]) > 10
```

Comment:
Don't we want to monitor thallium here?

Reply:
This is an exception group, so that they're not included in the p99 report

Reply:
Ok, sorry boss.